### PR TITLE
Runtime release checklist template update

### PR DESCRIPTION
**Pull request by muharem**
part of release process review initiative https://github.com/paritytech/cumulus/issues/1657
capturing the things discussed offline with release engineers and CGP team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-runtime.md [hunk 1]
```diff
@@ -6,21 +6,24 @@ title: Release Checklist for Runtime {{ env.VERSION }}
...
 ## Runtimes Release
 
 ### Codebase
 These checks should be performed on the codebase.
 
 - [ ] the [`spec_version`](https://github.com/paritytech/cumulus/blob/master/docs/release.md#spec-version) has been incremented since the
     last release for any native runtimes from any existing use on public (non-private/test) networks
-- [ ] previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are
-    removed for any public (non-private/test) networks
-- [ ] No migrations added in the last release that would need to be removed
```
Comment (muharem) [inline .github/ISSUE_TEMPLATE/release-runtime.md:18]: looks like duplication to me

File: .github/ISSUE_TEMPLATE/release-runtime.md [hunk 2]
```diff
@@ -6,21 +6,24 @@ title: Release Checklist for Runtime {{ env.VERSION }}
...
+- [ ] :crab: previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are removed for any public (non-private/test) networks
 - [ ] pallet and [extrinsic ordering](https://github.com/paritytech/cumulus/blob/master/docs/release.md#extrinsic-ordering--storage) as well as `SignedExtension`s have stayed
     the same. Bump `transaction_version` otherwise
 - [ ] the [benchmarks](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus) ran
 - [ ] the weights have been updated for any modified runtime logic
-- [ ] the various pieces of XCM config are sane
+- [ ] :crab: the new weights are sane, there are no significant (>20%) drops or rises with no reason
+- [ ] :crab: the various pieces of XCM config are sane
```
Comment (joepetrowski) [inline .github/ISSUE_TEMPLATE/release-runtime.md:26]: Maybe elaborate a little on "sane". Something like, "XCM was tested in integration testing and is compatible with the configurations and versions of relevant interlocutors, like the Relay Chain."

File: .github/ISSUE_TEMPLATE/release-runtime.md [hunk 3]
```diff
@@ -6,21 +6,24 @@ title: Release Checklist for Runtime {{ env.VERSION }}
...
-- [ ] previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are
-    removed for any public (non-private/test) networks
-- [ ] No migrations added in the last release that would need to be removed
+- [ ] :crab: previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are removed for any public (non-private/test) networks
 - [ ] pallet and [extrinsic ordering](https://github.com/paritytech/cumulus/blob/master/docs/release.md#extrinsic-ordering--storage) as well as `SignedExtension`s have stayed
     the same. Bump `transaction_version` otherwise
 - [ ] the [benchmarks](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus) ran
 - [ ] the weights have been updated for any modified runtime logic
-- [ ] the various pieces of XCM config are sane
+- [ ] :crab: the new weights are sane, there are no significant (>20%) drops or rises with no reason
```
Comment (gilescope) [inline .github/ISSUE_TEMPLATE/release-runtime.md:25]: > 50% for no reason (possibly more is appropriate). They do move about a lot.

File: .github/ISSUE_TEMPLATE/release-runtime.md (hunk 2 above)
Comment (gilescope) [reply .github/ISSUE_TEMPLATE/release-runtime.md:26]: I think we should be building out our integration tests to verify the things that we want to check for. Or at least enumerating the things that should be checked.

File: .github/ISSUE_TEMPLATE/release-runtime.md [hunk 4]
```diff
@@ -6,21 +6,24 @@ title: Release Checklist for Runtime {{ env.VERSION }}
...
+- [ ] :crab: the various pieces of XCM config are sane
 
 ### On the release branch
```
Comment (gilescope) [inline .github/ISSUE_TEMPLATE/release-runtime.md:29]:
```suggestion
### On the release branch

- [ ] :crab: Doublecheck that the release branch is not pointing to master for cumulus/polkadot/substrate.
```

Comment (gilescope) [reply .github/ISSUE_TEMPLATE/release-runtime.md:29]: At some point we can take out this check but for now this is something to be checked as it's not yet automated.

**Review by gilescope — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release-runtime.md (hunk 2 above)
Comment (muharem) [reply .github/ISSUE_TEMPLATE/release-runtime.md:26]: @gilescope agree, we are discussing it here https://github.com/paritytech/cumulus/issues/1682

Comment (muharem) [reply .github/ISSUE_TEMPLATE/release-runtime.md]: @joepetrowski update to - "XCM config is compatible with the configurations and versions of relevant interlocutors, like the Relay Chain."
There is a check below for integration tests

File: .github/ISSUE_TEMPLATE/release-runtime.md (hunk 3 above)
Comment (muharem) [reply .github/ISSUE_TEMPLATE/release-runtime.md]: updated

**Review by joepetrowski — APPROVED** [review]

**Review by chevdor — APPROVED** [review]

**Review by EgorPopelyaev — APPROVED** [review]

**Pull request merged**